src/DynamoPackages/Package.cs

```diff
@@ -113,6 +114,60 @@ public bool MarkedForUninstall
             internal set { markedForUninstall = value; RaisePropertyChanged("MarkedForUninstall"); }
         }
 
+        public bool EnableOldMarkedForUnistallState
```

Comment:
seems that wpf can only bind to public properties...:(
If we want to have it under a debug mode first....we could just tag these temporary public methods...that they should not be used ..?

Reply:
yeah, one thing we've done in the past is to mark with obsolete immediately - it's confusing, but it let's people know to be wary.

Reply:
I don't see any harm to removing an obsolete method that we've explicitly called out as DO NOT USE.

Reply:
> I don't see any harm to removing an obsolete method that we've explicitly called out as DO NOT USE.

Yes, that one I agree. I am curious about the case that we obsoleted something but decided that we want to keep it in the end, do we simply remove the obsolete tag?

Reply:
yep :) - Again, what cause concerns you? if no one is using the member for fear it might be removed, then what break could keeping it cause?

Reply:
👍